site/src/components/ComponentsList.js

```diff
@@ -147,6 +147,20 @@ const ComponentsList = () => {
         width: 280,
       },
     },
+    {
+      name: 'Header',
+      text: 'Header is a wrapper component for all navigation elements.',
+      linkboxAriaLabel: 'Header component',
+      linkAriaLabel: 'Go to the Header component page',
+      href: '/components/header',
+    },
+    {
+      name: 'HeaderUniversalBar',
```

Comment:
I'm not sure do we need separate pages for HeaderUniversalBar.
Would it be enough to have it as a section in the Header component usage and code page?

Reply:
I think that's fair. My main concern was that there would be too much content in usage page and it would get messy in code page. But looking at the lack of content on -UniversalBar doc page now, I don't think these Header- components will need too much explaining 😄 And Navigation component works as an example for a pretty readable code page properties section as well. So sure, I'll merge -UniversalBar into Header.

Reply:
Perhaps I should remove HeaderUniversalBar story as well as it can't be used alone? It would be covered in Header.stories.